problem/0000-rfc-template.md

```diff
@@ -0,0 +1,71 @@
+- Contribution Name: Thrift Support
+- Implementation Owner: Peter Novotank <peter.novotnak@reddit.com>, Linkerd2 Authors
+- Start Date: 6/3/2020
+- Target Date: 6/15/2020
+- RFC PR: [linkerd/rfc#9](https://github.com/linkerd/rfc/pull/9)
+- Linkerd Issue: [linkerd/linkerd2#0000](https://github.com/linkerd/linkerd2/issues/0000)
+- Reviewers: (Who should review the code deliverable? ex.@olix0r)
+
+# Summary
+
+[summary]: #summary
+
+Linkerd2-proxy and control plane support for the Thrift protocol.
+
+# Problem Statement (Step 1)
+
+[problem-statement]: #problem-statement
+
+- Add protocol support to the linkerd2-proxy for at least TFramedTransport and TBinaryProtocol 
+- Support routing, retries and metrics for Thrift.
```

Comment:
Response classification is fairly easy in HTTP since we assume (by default) that any response with status code 5XX is a failure and any other response code is a success.  Thrift does not have status codes like this so it is less clear how to determine if a response should be considered successful or not.  How do you go about calculating the success rate of a thrift service?